tests/cover/test_conjecture_minimizer.py

```diff
@@ -17,15 +17,18 @@
 
 from __future__ import division, print_function, absolute_import
 
+from random import Random
```

Comment:
Cool - not something to address here, but is there documentation about the minimizer which mentions that?